docs/UserGuide.md

```diff
@@ -235,6 +220,46 @@ _Details coming soon ..._
 
 --------------------------------------------------------------------------------------------------------------------
 
+## Fields
```

Comment:
Should there be valid input for the find method? Will it be under this section as well?